packages/docs/developer-resources/contractkit/migrating-to-contractkit-v1.md

```diff
@@ -0,0 +1,84 @@
+# Migrating to ContractKit v1.0.0
+
+cLabs recently released ContractKit version 1.0.0. This document explains the key differences and shows you how you can start using the updated package.
+
+If you are using a previous version of ContractKit, you can continue using that version and you will only need to make the following changes when you upgrade.
+
+The main benefit of using the new version include:
+ - Reduced bundle size
+ - Better typescript support
+ - Improved maintenance by making it easier to use other libraries
+
+## ContractKit packages
+
+ContractKit is now a [suite of packages](https://github.com/celo-org/celo-monorepo/tree/master/packages/sdk).
+
+### Main packages
+
+ - `Connect` depends on the `web3` package and allows you to specify the connection provider and contains some configuration information for signing transactions.
```

Comment:
> This package basically handles how we communicate to the our chain nodes. It wraps the `web3` library and has its own `rpcCaller` class, to make custom calls to the node. It's the layer in charge of knowing how and which parameters are added by Celo, connect to the node, build the message, sent it and handle those responses.

Not sure how much of this you wanted to add, but i copied over Ponti's verbiage from the original [PR description](https://github.com/celo-org/celo-monorepo/pull/4790) which I think has some useful language.

Reply:
awesome, thanks!